deployments/UnityMixpanel/ProjectSettings/GraphicsSettings.asset

```diff
@@ -1,74 +0,0 @@
-%YAML 1.1
```

Comment:
I discovered that Unity will re-create them, and I don't see a need to keep them in version control since we've haven't customized them.